services/src/modules/storage/s3.ts

```diff
@@ -1,4 +1,5 @@
 import * as AWS from 'aws-sdk';
+import { S3 } from 'aws-sdk';
```

Comment:
This is exactly the same as using `AWS.S3` like everywhere else in the file, so you should either use the same or change everywhere else in the file to use the "new" import and remove the old import (since there is no usage of `AWS` in the file that is not `AWS.S3`)

Reply:
Autocomplete... I didn't notice. I'll leave the `import { S3 } from 'aws-sdk'`. It's more specific.